functions.py

```diff
@@ -1016,6 +1016,13 @@ def ApplyUserEdit(form, session):
             return
         OriginalPriv = OriginalPriv[0][0]
         if int(Privilege) > OriginalPriv:
+            RAPLog(session f"Attempted to set their own privilege to {Privilege}: (Original: {OriginalPriv})") # Snitch on them
```

Comment:
u cant even edit urself???

Reply:
💀 most useless feature of 2022

Reply:
you can't edit yourself but you can give other people privileges higher than your own. this prevents that

Reply:
You can edit yourself but not give yourself higher privileges

Reply:
and there's no point in "snitching"